content/en/logs/error_tracking/excluding_logs.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Excluding Logs from Error Tracking
+kind: documentation
+disable_toc: false
+---
+
+## Overview
+
+{{< img src="logs/error_tracking/exclusion_filters.png" alt="Exclusion filters list" style="width:100%" >}}
+
+Exclusion filters prevent Error Tracking from processing logs matching a given query, lowering issue volume and reducing noise. 
+
+For example, you can use an exclusion filter to ignore logs from the staging environment (see [Filter setup](#filter-setup)).
+
+### Scope
+
+An Error Tracking exclusion filter only impacts which issues are processed by Error Tracking. To filter logs from Log Management, see [Create a log pattern exclusion filter][3]. To exclude logs from being sent to Datadog, see [Filter logs][2].
+
+### Ordering
+
+The filters in the **Exclusion Filters** table are applied in order. For example, creating a filter on the query `env:staging host:example-host` has no effect if an `env:staging` filter already exists.
+
+Filters can be reordered in the table by clicking on the right side of each row and dragging up or down.
+
+## Filter setup
+
+This example creates an exclusion filter for logs matching `env:staging`, but the steps are similar for any query.
+
+{{< img src="logs/error_tracking/configured_exclusion_filter.png" alt="Exclusion filter configuration for ignoring staging logs" style="width:100%" >}}
+
+1. In Datadog, go to [Logs > Error Tracking > Settings > Exclusion Filters][1].
+2. Click **Add Exclusion Filter**.
+3. In the **Name** field, enter "Staging logs".
+4. In the **Define exclusion query** field, type "env", then choose the `env` tag from the autocomplete suggestions to view available tag values. Choose `env:staging`.
```

Comment:
```suggestion
4. In the **Define exclusion query** field, type "env", then choose the `env` tag from the autocomplete suggestions to view available tag values. Choose `env:staging`.

   {{< img src="logs/error_tracking/configured_exclusion_filter.png" alt="Exclusion filter configuration for ignoring staging logs" style="width:100%" >}}
```

Reply:
Just an editorial suggestion, but I thought it might be helpful to move the screenshot to be more in-context.

Reply:
Thank you! For query examples I've been putting the screenshot at the top because I don't think most people even want to read the instructions at all ... so the instructions are more for folks who can't see images or are wanting a little more help making that initially displayed end result happen.